storage/badger/chunkDataPacks.go

```diff
@@ -49,7 +50,8 @@ func NewChunkDataPacks(collector module.CacheMetrics, db *badger.DB, byChunkIDCa
 }
 
 func (ch *ChunkDataPacks) Store(c *flow.ChunkDataPack) error {
-	err := operation.RetryOnConflictTx(ch.db, transaction.Update, ch.byChunkIDCache.PutTx(c.ChunkID, c))
+	sc := toStoredChunkDataPack(c)
```

Comment:
chunk data pack is converted internally to a `StoredChunkDataPack` for maintaining on storage.